README.md

```diff
@@ -104,4 +104,4 @@ When developing from within the mono-repo, the libraries can be individually ins
 
 To install the base dependencies, run `pip install -e .` from the mono-repo root. Development dependencies can be installed with `pip install -e .[dev-dependencies]`
 
-To install the library dependencies, run, for example, `pip install -e src/web` to install `BL_Python.web`. Similar to the mono-repo, development dependencies can be installed with `pip install -e src/web[dev-dependencies]`.
\ No newline at end of file
+To install the library dependencies, run, for example, `pip install -e src/web` to install `BL_Python.web`. Similar to the mono-repo, development dependencies can be installed with `pip install -e src/web[dev-dependencies]`. To install all libraries, you can run `./install_all.sh`.
```

Comment:
Looks good. Just a quick clarification point. the `install_all.sh` script installs the base dependencies, the mono-repo development dependencies, and the library dependencies, but not the development dependencies for each library, correct? 

When I read line 107 and I see we talk about library dependences, then library specific developement dependencies, followed by a script called `install_all.sh` I might think the library specific development dependencies will also be installed.

Reply:
You're correct. The reason I added only `.[dev-dependencies]` and not each of the individual library dev-dependencies is because 1. only the monorepo and `BL_Python.web` define them and 2. they are equivalent. I hemmed and hawed on this and am leaning toward not including `optional-dependencies` in the individual libraries, which sets the expectation that development on any of them means you must develop in the monorepo. This is likely the path of least resistance regardless, given this is a monorepo.